FABulous/fabric_definition/Fabric.py

```diff
@@ -252,10 +260,16 @@ def __repr__(self) -> str:
         return fabric
 
     def getTileByName(self, name: str) -> Tile | None:
-        return self.tileDic.get(name)
+        ret = self.tileDic.get(name)
```

Comment:
I think you will need the `.get(name, None)`; otherwise, it will raise an error on the first get miss

Reply:
None is the default for get() on dict

https://www.w3schools.com/python/ref_dictionary_get.asp